src/Showtime.jsx

```diff
@@ -70,5 +70,5 @@ export default function Showtime({
         }
     }, [status, previousStatus, onHidden, onShowing]);
 
-    return isMounted ? children(ref) : null;
+    return isMounted ? cloneElement(Children.only(children), { ref }) : null;
```

Comment:
`Children.only` ensures that there is exactly one passed child to the `Showtime` prop, will throw an error otherwise.